build-logic/src/main/kotlin/kotlinx/io/conventions/kotlinx-io-multiplatform.gradle.kts

```diff
@@ -80,6 +80,12 @@ kotlin {
                         group("androidNative")
                     }
                 }
+
+                group("nonAndroid") {
```

Comment:
Am I right that `nonAndroid` is now actually a synonym for `posix`?

Reply:
Well, the only POSIX certified target down there is macOs :)

Reply:
But for somehow we use posix API for all of them ))

Reply:
It's just that for me, JVM is also a non-Android, so this name was confused at first

Reply:
But Android is as POSIX as Linux

Reply:
Perhaps, `commonizedAsPosix`?

Reply:
Yet another strenuous attempt to give a better name:
- `nativeNonAndroid`
- rename `nonApple` to `nativeNonApple`

WDYT?